src/xdocs/config_javadoc.xml

```diff
@@ -396,12 +396,12 @@ public int field;
     </section>
 
     <section name="JavadocMethod">
+      <p>Since Checkstyle 3.0</p>
       <subsection name="Description" id="JavadocMethod_Description">
-        <p>Since Checkstyle 3.0</p>
         <p>
           Checks the Javadoc of a method or constructor. By default,
-          does not check for unused throws. To allow documented
-          <code>java.lang.RuntimeException</code>s that are not
```

Comment:
Rewritten since it violates rule "a space is required after the code tag" from XDocsJavaDocsCheck.